code/modules/modular_computers/ntos/files.dm

```diff
@@ -87,4 +87,190 @@
 	if(!istype(F))
 		return FALSE
 	var/datum/computer_file/C = F.clone(0)
-	return disk_to.store_file(C)
\ No newline at end of file
+	return disk_to.store_file(C)
+
+// Generic file storage interface
+/datum/file_storage
+	var/name = "Generic File Interface"
+	var/datum/extension/interactive/ntos/os
+/datum/file_storage/New(ntos)
```

Comment:
What's with the weird whitespace use?

Reply:
it's a bit of a haze, but i tihnk i didn't like how there's so many empty functions
does look nicer with whitespaces tho, changed